client/src/items/Computer.ts

```diff
@@ -0,0 +1,66 @@
+import { ItemType } from '../../../types/Items'
+import store from '../stores'
+import Item from './Item'
+import Network from '../services/Network'
+import { openComputerDialog } from '../stores/ComputerStore'
+
+export default class Computer extends Item {
+  id?: string
+  currentUsers = new Array<string>()
```

Comment:
you can actually just do `currentUsers: string[] = []`, no need to explicitly call `new Array` to create an empty array

Reply:
Changed to `currentUsers: string[] = []`